vulture/core.py

```diff
@@ -395,6 +395,9 @@ def visit_ClassDef(self, node):
         else:
             self._define(
                 self.defined_classes, node.name, node, ignore=_ignore_class)
+    
+    def visit_Constant(self, node):
```

Comment:
Is this method really called at all? node.s is accessed in visit_Str and I doubt that all constants have this field.

Reply:
> Is this method really called at all?

I just tried a basic example:

```python
class CustomNodeVisitor(ast.NodeVisitor):
    def scan(self, code):
        self.visit(ast.parse(code))

    def visit_Constant(self, node):
        print("I'm visiting a `Constant` node! \o/")

code = """\
import this
username = "RJ722"
"""

c = CustomNodeVisitor()
c.scan(code)
```

which has the following output:
```
I'm visiting a `Constant` node! \o/
```

so I guess it is being called.

>  I doubt that all constants have this field.

Yeah, maybe we can try something like:

```python
def visit_Constant(self, node):
    if getattr(node, 's'):
        self.visit_Str(node)
```

However, I'm not sure that this is correct!

Reply:
I edited my previous comment to simplify the test case:

Initially, it was:

```python
import this
username = 'RJ722'
```

which has a redundant import statement and an unused variable. We can just use:

```
'RJ722'
```